### ISSUE TITLE
SmartCity-Accessibility #166 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#116]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 71 lines...]
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 55 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/jxMapsFunctionality/MapIntegeration.java: /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/jxMapsFunctionality/MapIntegeration.java uses or overrides a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/jxMapsFunctionality/MapIntegeration.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[10,42] cannot find symbol
  symbol:   class ScoreNotInRangeException
  location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Review.java:[6,42] cannot find symbol
  symbol:   class ScoreNotInRangeException
  location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Score.java:[3,42] cannot find symbol
  symbol:   class ScoreNotInRangeException
  location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Score.java:[21,44] cannot find symbol
  symbol:   class ScoreNotInRangeException
  location: class smartcity.accessibility.socialnetwork.Score
[INFO] 4 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13.156 s
[INFO] Finished at: 2016-12-29T16:28:57+00:00
[INFO] Final Memory: 26M/174M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[10,42] cannot find symbol
[ERROR] symbol:   class ScoreNotInRangeException
[ERROR] location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Review.java:[6,42] cannot find symbol
[ERROR] symbol:   class ScoreNotInRangeException
[ERROR] location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Score.java:[3,42] cannot find symbol
[ERROR] symbol:   class ScoreNotInRangeException
[ERROR] location: package smartcity.accessibility.exceptions
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/Score.java:[21,44] cannot find symbol
[ERROR] symbol:   class ScoreNotInRangeException
[ERROR] location: class smartcity.accessibility.socialnetwork.Score
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [KoralChapnik] 97322ffebed4d15042b15d1d8999c7ae83ede0c5 - removed ScoreNotInBountdException #97

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/166/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
